scene/animation/animation_mixer.cpp

```diff
@@ -866,11 +879,13 @@ bool AnimationMixer::_update_caches() {
 					}
 				}
 
-				if (!skip_update_mode_warning && was_continuous != track_value->is_continuous) {
-					WARN_PRINT_ONCE_ED("AnimationMixer: '" + String(E) + "', Value Track: '" + String(path) + "' has different update modes between some animations which may be blended together. Blending prioritizes UpdateMode.UPDATE_CONTINUOUS, so the process treats UpdateMode.UPDATE_DISCRETE as UpdateMode.UPDATE_CONTINUOUS with InterpolationType.INTERPOLATION_NEAREST.");
+				if (check_angle_interpolation) {
+					if (!skip_update_mode_warning && was_continuous != track_value->is_continuous) {
+						WARN_PRINT_ED("AnimationMixer: '" + String(E) + "', Value Track: '" + String(path) + "' has different update modes between some animations which may be blended together. Blending prioritizes UpdateMode.UPDATE_CONTINUOUS, so the process treats UpdateMode.UPDATE_DISCRETE as UpdateMode.UPDATE_CONTINUOUS with InterpolationType.INTERPOLATION_NEAREST.");
```

Comment:
Ah, it may have gone wrong in the process because we repeated the Rebase. However, this Warning conflicts with #86629 and will eventually be removed. So this PR assumes #86629.